### ISSUE TITLE
0.7.4 Fails with missing `read:org` scope

### ISSUE THREAD

**Issue by aricallen** [mszostok/codeowners-validator#224]
**Description**

We updated to 0.7.4 and now see an error: `level=fatal msg="while checking if 'owners' checker is satisfied: missing scopes: \"read:org\""`

Downgrading back to 0.7.2 fixes the issue.

**Expected result**

Updating to the latest patch version should work the same without required changes.

**Actual result**

The check failed.

**Steps to reproduce**

Upgrade to 0.7.4 without changing anything else.

**Troubleshooting**

Downgrading to 0.7.2 fixed the issue.

**Comment by ov7a** [comment]
Workaround: include the action twice

```yml

      - name: GitHub CODEOWNERS Validator
        uses: mszostok/codeowners-validator@v0.7.4
        if: github.event.pull_request.head.repo.full_name == github.repository
        with:
          checks: "files,duppatterns,syntax"
          experimental_checks: "notowned,avoid-shadowing"

      # downgrading to 0.7.2 for owners check to avoid issues with token permissions check,
      # see https://github.com/mszostok/codeowners-validator/issues/224
      - name: GitHub CODEOWNERS Validator
        uses: mszostok/codeowners-validator@v0.7.2
        if: github.event.pull_request.head.repo.full_name == github.repository
        with:
          checks: "owners"
          github_access_token: "${{ secrets.SPECIAL_GITHUB_ACCESS_TOKEN }}"
          owner_checker_owners_must_be_teams: true
```

**Comment by marcodicro-dp** [comment]
Any chance to fix this so we can go back to using the latest version?
